### PR TITLE
add mesos + marathon example configs

### PR DESCRIPTION
**Pull request by siggy**
depends on https://github.com/BuoyantIO/linkerd-viz/pull/22

### CODE REVIEW

File: mesos-marathon/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Example configs for Mesos + Marathon
```
Comment (klingerf) [inline mesos-marathon/README.md]: Mind providing a short intro of what this directory contains? Something like:

> This directory provides a [docker-compose.yml]() file that can be used to run [Mesos]() and [Marathon](). It also provides two Marathon config files for running Linkerd and a sample web app, once the docker cluster is initialized.

File: mesos-marathon/linkerd-marathon.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "protocol": "tcp",
+      "name": "admin"
+    },
+    {
+      "port": 4140,
+      "protocol": "tcp",
+      "name": "proxy"
+    }
+  ],
+  "cmd": "echo \"{\\\"admin\\\":{\\\"port\\\":9990},\\\"telemetry\\\":[{\\\"kind\\\":\\\"io.l5d.prometheus\\\"}],\\\"namers\\\":[{\\\"kind\\\":\\\"io.l5d.marathon\\\",\\\"host\\\":\\\"localhost\\\",\\\"port\\\":8080,\\\"experimental\\\":true}],\\\"routers\\\":[{\\\"protocol\\\":\\\"http\\\",\\\"servers\\\":[{\\\"port\\\":4140,\\\"ip\\\":\\\"0.0.0.0\\\"}],\\\"dtab\\\":\\\"/marathonId=>/#/io.l5d.marathon;/svc=>/$/io.buoyant.http.domainToPathPfx/marathonId;\\\",\\\"label\\\":\\\"linkerd_proxy\\\"}]}\" | /io.buoyant/linkerd/0.9.0/bundle-exec -log.level=DEBUG -com.twitter.finagle.tracing.debugTrace=true -- -"
```
Comment (adleong) [inline mesos-marathon/linkerd-marathon.json]: Should we also check-in the source config this was generated from? (and instructions for how to generate it)

File: mesos-marathon/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Get Docker IP address
```
Comment (klingerf) [inline mesos-marathon/README.md]: Maybe provide some context here for why the DOCKER_IP variable is required.

**Comment by klingerf** [comment]
I was able to run through the README almost to completion, but I ran into an issue with the Marathon container crashing after I had deployed linkerd + webapp. Not exactly sure what's going on with that, but otherwise looks good, and this is sweet.

**Comment by siggy** [comment]
@klingerf yeah, i noticed that marathon issue as well. it was losing connection to zk. i bumped it all the way down to `v0.15.7` and it started failing in a different way, settled on `v1.3.0` which has not crashed, yet.

**Pull request merged**